### ISSUE TITLE
Validator logs overflow screen

### ISSUE THREAD

**Issue by nathanleclaire** [workbenchapp/solana-workbench#179]
not a big deal, but to track ... validator logs overflow the screen .. it should probably work more like the embedded program logs on accounts page (with amman at play, I'm tempted to say we should just embed Amman logs there instead ... no idea what that would mean for mainnet etc though)

https://user-images.githubusercontent.com/1476820/177882678-99f8e1ca-765c-48d6-8e9f-7e1ccf891330.mov

**Comment by SvenDowideit** [comment]
yeah, i was expecting to have a scroll bar like on the program changes
